<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="BorderlessWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="mainLayout" stretch="1">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="hPlots" stretch="1,0">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTabWidget" name="twReal">
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="tabPanel" name="twSim">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="SimulationFrame" name="tSim">
             <attribute name="title">
              <string>Simulation</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="twAberr">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="SourceFrame" name="tMicroscope">
             <attribute name="title">
              <string>Microscope</string>
             </attribute>
            </widget>
            <widget class="AberrationFrame" name="tAberr">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="title">
              <string>Aberrations</string>
             </attribute>
            </widget>
            <widget class="InelasticFrame" name="tInelastic">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="title">
              <string>Inelastic</string>
             </attribute>
            </widget>
            <widget class="IncoherenceFrame" name="tIncoherence">
             <attribute name="title">
              <string>Incoherence</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="twMode">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="usesScrollButtons">
             <bool>false</bool>
            </property>
            <widget class="TemFrame" name="tTem">
             <attribute name="title">
              <string>TEM</string>
             </attribute>
            </widget>
            <widget class="StemFrame" name="tStem">
             <attribute name="title">
              <string>STEM</string>
             </attribute>
            </widget>
            <widget class="CbedFrame" name="tCbed">
             <attribute name="title">
              <string>CBED</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="tabPanel" name="twRun">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tRun">
             <attribute name="title">
              <string>Run</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Iterations</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="EditUnitsBox" name="edtIterations"/>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnCancelSim">
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnStartSim">
                <property name="text">
                 <string>Sim</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuParameters">
     <property name="title">
      <string>Config...</string>
     </property>
     <addaction name="actionImport_parameters"/>
     <addaction name="actionExport_parameters"/>
     <addaction name="separator"/>
     <addaction name="actionImport_default"/>
     <addaction name="actionExport_default"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuParameters"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <widget class="QMenu" name="menuInelastic">
     <property name="title">
      <string>Inelastic</string>
     </property>
     <addaction name="actionThermal_scattering"/>
     <addaction name="actionPlasmons"/>
    </widget>
    <addaction name="actionGeneral"/>
    <addaction name="menuInelastic"/>
    <addaction name="actionSet_area"/>
    <addaction name="actionAberrations"/>
    <addaction name="separator"/>
    <addaction name="actionSimulate_EW"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOpenCL"/>
    <addaction name="actionTheme"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuInput"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar::item{border: 0px}</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open structure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSimulate_EW">
   <property name="text">
    <string>Simulate</string>
   </property>
  </action>
  <action name="actionSimulate_Image">
   <property name="text">
    <string>Simulate Image</string>
   </property>
  </action>
  <action name="actionSet_area">
   <property name="text">
    <string>Area</string>
   </property>
  </action>
  <action name="actionOpenCL">
   <property name="text">
    <string>OpenCL</string>
   </property>
  </action>
  <action name="actionGeneral">
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actionExport_2">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionImportSettings">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExportSettings">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport_2">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport_3">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExport_parameters">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport_parameters">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionAberrations">
   <property name="text">
    <string>Aberrations</string>
   </property>
  </action>
  <action name="actionImport_3">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionThermal_scattering">
   <property name="text">
    <string>Phonons</string>
   </property>
  </action>
  <action name="actionNative">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Native</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actionImport_default">
   <property name="text">
    <string>Import default</string>
   </property>
  </action>
  <action name="actionExport_default">
   <property name="text">
    <string>Export default</string>
   </property>
  </action>
  <action name="actionShow_default">
   <property name="text">
    <string>Show default config</string>
   </property>
  </action>
  <action name="actionPlasmons">
   <property name="text">
    <string>Plasmons</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>tabPanel</class>
   <extends>QTabWidget</extends>
   <header>controls/tabpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditUnitsBox</class>
   <extends>QLineEdit</extends>
   <header>controls/editunitsbox.h</header>
  </customwidget>
  <customwidget>
   <class>SimulationFrame</class>
   <extends>QWidget</extends>
   <header>frames/simulationframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TemFrame</class>
   <extends>QWidget</extends>
   <header>frames/temframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StemFrame</class>
   <extends>QWidget</extends>
   <header>frames/stemframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CbedFrame</class>
   <extends>QWidget</extends>
   <header>frames/cbedframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AberrationFrame</class>
   <extends>QWidget</extends>
   <header>frames/aberrationframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BorderlessWindow</class>
   <extends>QMainWindow</extends>
   <header>controls/borderlesswindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InelasticFrame</class>
   <extends>QWidget</extends>
   <header>frames/inelasticframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SourceFrame</class>
   <extends>QWidget</extends>
   <header>frames/sourceframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IncoherenceFrame</class>
   <extends>QWidget</extends>
   <header>frames/incoherenceframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
